migrations/20240117071311_create_function_tx_join_group_smart_contract_event.ts

```diff
@@ -0,0 +1,22 @@
+import { Knex } from 'knex';
+
+export async function up(knex: Knex): Promise<void> {
+  await knex.schema.raw(`
+    CREATE OR REPLACE FUNCTION public.transaction_join_group_smart_contract_event(contractaddress character varying)
```

Comment:
hmm câu này ko có limit, có ổn ko? em thử với mấy cái contract to chưa

Reply:
em thử thấy oke, không như 2 câu query bên account phải cho cả limit vào mới chạy được

Reply:
mà thôi để em thêm limit vào cho nhanh luôn